src/main/webapp/project/NewProjectDocuments.js

```diff
@@ -104,15 +104,21 @@ export const NewProjectDocuments = hh(class NewProjectDocuments extends Componen
     }
 
     let documents = this.props.files;
-
     let errors = false;
-
+    let errorMessage = ''
+    if(!this.props.errors.attestation && this.props.generalError) {
+      errorMessage = 'Please check previous steps';
+    } else if (this.props.submitError) {
+      errorMessage =  'Something went wrong. Please try again.';
+    } else {
+      errorMessage = 'Please complete all required fields';
```

Comment:
See lines 112 and 114 here - are they swapped? It seems to be the last `else` condition would have the `'Something went wrong. Please try again.'` message.

Reply:
They are not swapped, If we have an error after we try to submit the changes then we have to show 'Something went wrong. Please try again.'.
 If not, we have validations error, then we show 'Please complete all required fields'